lib/doorkeeper/orm/active_record.rb

```diff
@@ -20,11 +20,8 @@ def self.initialize_application_owner!
       end
 
       def self.check_requirements!(_config)
-        if ::ActiveRecord::Base.connected? &&
-           ::ActiveRecord::Base.connection.table_exists?(
-             Doorkeeper::Application.table_name
-           )
-          unless Doorkeeper::Application.column_names.include?("scopes")
+        if ::ActiveRecord::Base.connected? && Doorkeeper::Application.table_exists?
+          if Doorkeeper::Application.column_names.exclude?("scopes")
             migration_path = '../../../generators/doorkeeper/templates/add_scopes_to_oauth_applications.rb'
             puts <<-MSG.squish
 [doorkeeper] Missing column: `oauth_applications.scopes`.
```

Comment:
This warning message was useful while upgrading to doorkeeper 3. I'll ditch it altogether for v4.